templates/vulnerability/embedded.html

```diff
@@ -82,9 +75,9 @@
     getWorkerUrl: function (workerId, label) {
       return `data:text/javascript;charset=utf-8,${encodeURIComponent(`
         self.MonacoEnvironment = {
-          baseUrl: 'https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.14.3/min/'
+          baseUrl: "{{ url_for('frontend.serve_static', path='monaco/min') }}"
         };
-        importScripts('https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.14.3/min/vs/base/worker/workerMain.js');`
+        importScripts("{{ url_for('frontend.serve_static', path='monaco/min/vs/base/worker/workerMain.js') }}");`
       )}`;
     }
   };
```

Comment:
can you remove the whole `window.MonacoEnviroment` part (including comment). As monaco is now served directly, this is a) not needed anymore and b) breaks some functionality